src/baseStyleUtils.js

```diff
@@ -0,0 +1,66 @@
+const css = (strings, ...expressions) => {
+  let str = ''
+  strings.forEach((string, i) => {
+    str += string + (expressions[i] || '')
+  })
+
+  return str
+}
+
+const generateBaseStyles = ({ breakpoint = '40em' }) => css`
+.super-responsive-table__th_pseudo {
+  display: none;
+}
+
+@media screen and (max-width: ${breakpoint}) {
+  .super-responsive-table__table,
+  .super-responsive-table__thead,
+  .super-responsive-table__tbody,
+  .super-responsive-table__th,
+  .super-responsive-table__td,
+  .super-responsive-table__tr {
+    display: block;
+  }
+
+  .super-responsive-table__thead {
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+  }
+
+  .super-responsive-table__td_pivoted {
+    position: relative;
+    margin-left: 50%;
```

Comment:
Because `padding` is often used to set table padding, use `margin-left` instead of `padding-left`, so that user-defined styles won't interfere with the base styles.